modules/insights-operator-advisor-overview.adoc

```diff
@@ -8,9 +8,17 @@
 
 You can use Insights Advisor to assess and monitor the health of your {product-title} clusters. Whether you are concerned about individual clusters, or with your whole infrastructure, it is important to be aware of the exposure of your cluster infrastructure to issues that can affect service availability, fault tolerance, performance, or security.
 
-Using cluster data collected by the Insights Operator, Insights repeatedly compares that data against a library of _recommendations_. Each recommendation is a set of cluster-environment conditions that can leave {product-title} clusters at risk. The results of the Insights analysis are available in the Insights Advisor service on Red Hat Hybrid Cloud Console. In the Console, you can perform the following actions:
+If the cluster has the Deployment Validation Operator installed the recommendations also highlight workloads whose configuration could lead to cluster health issues.
 
-* See clusters impacted by a specific recommendation.
+The results of the Insights analysis are available in the Insights Advisor service on {hybrid-console}. In the Console, you can perform the following actions:
```

Comment:
Is `Console` a short form for {hybrid-console}? Is yes, we can have an attribute for it. I would recommend reusing the hybrid-console itself.